optaplanner-docs/src/main/docbook/en-US/Chapter-Benchmarking_and_tweaking/Chapter-Benchmarking_and_tweaking.xml

```diff
@@ -952,6 +952,16 @@
           If it's higher, it will be automatically decreased.</para>
         </note>
 
+        <warning>
```

Comment:
I'd make it a note. I still presume your laptop type is the exception to the rule.